### PR TITLE
chore: Modify create event payload to use new schema

### PR DESCRIPTION
**Pull request by dav1do**
Linear WS1-1485 (also does part of WS1-1475 in the refactoring and tackles [WS1-1456](https://linear.app/3boxlabs/issue/WS1-1456/update-ceramic-tests-to-use-feed-and-events-endpoints) in addressing subscribe payload differences).

`ceramic-one` supports `id/data` as the event payload for [input and output now](https://github.com/ceramicnetwork/rust-ceramic/pull/269), so we update to the preferred format, and swap the /subscribe calls to use the endpoint that allows querying events by an interest (in our case, model ID).

I added status code asserts on most of the HTTP calls and refactored some code into the utils file and shared it in `get-events` and `events` (renamed to `sync-events` to make it easier to run just on its own). I updated to nightly packages to get the fix from [js-ceramic PR 3133](https://github.com/ceramicnetwork/js-ceramic/pull/3133) which required changing the update script as not all packages support nightly. Finally, I added we build (`tsc`) before running the tests so no one else misses that like I did.

### CODE REVIEW

**Comment by dav1do** [comment]
When this merges, the rust-ceramic PR should be g2g: https://github.com/ceramicnetwork/rust-ceramic/pull/272

File: suite/src/utils/rustCeramicHelpers.ts [hunk 1]
```diff
@@ -1,26 +1,60 @@
...
+  return modelEvents
+}
+
+export async function getEventData(url: string, eventId: string, log = false) {
+  let response = await fetch(url + `/ceramic/events/${eventId}`)
+  if (log) {
+    console.log(response)
+  }
+  return response
+}
```
Comment (smrz2001) [inline suite/src/utils/rustCeramicHelpers.ts:60]: Mind adding a new line at the end?

File: suite/src/utils/rustCeramicHelpers.ts [hunk 2]
```diff
@@ -1,26 +1,60 @@
...
+
+  for (let i = 0; i < count; i++) {
+    const eventId = generateRandomEventId(modelID, network, networkOffset)
+    const event = generateRandomEvent(eventId)
+    modelEvents.push(event)
+  }
+  return modelEvents
+}
+
+export async function getEventData(url: string, eventId: string, log = false) {
```
Comment (smrz2001) [inline suite/src/utils/rustCeramicHelpers.ts:54]: Should this function be moved here or remain in the test file?

**Review by smrz2001 — APPROVED** [review]
LGTM, just a couple of minor questions/comments.

File: suite/src/utils/rustCeramicHelpers.ts (hunk 2 above)
Comment (dav1do) [reply suite/src/utils/rustCeramicHelpers.ts]: yeah, I moved it back to the test. I thought I was going to use it in the sync tests originally but it wasn't necessary.

**Pull request merged**